shopfloor/tests/test_zone_picking_base.py

```diff
@@ -12,6 +12,24 @@ def setUpClassVars(cls, *args, **kwargs):
         cls.wh = cls.profile.warehouse_id
         cls.picking_type = cls.menu.picking_type_ids
 
+    @classmethod
+    def setUpClassUsers(cls):
+        super().setUpClassUsers()
+        Users = (
+            cls.env["res.users"]
+            .sudo()
+            .with_context({"no_reset_password": True, "mail_create_nosubscribe": True})
```

Comment:
use kw args otherwise you wipe the ctx and lose tracking_disable which is set in base test class :wink: